viewer/packages/tools/src/Measurement/MeasurementControls.ts

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright 2022 Cognite AS
+ */
+
+import { Cognite3DViewer } from '@reveal/core';
+import * as THREE from 'three';
+import { MeasurementGizmo } from './MeasurementGizmo';
+import { MeasurementLabel } from './MeasurementLabel';
+import { Measurement } from './Measurement';
+import { MeasurementLineOptions } from './types';
+import { MeasurementDistance } from './MeasurementDistance';
+
+export class MeasurementControls {
+  private readonly _viewer: Cognite3DViewer;
+  private readonly _measurementGizmo: MeasurementGizmo;
+  private readonly _measurementLabel: MeasurementLabel;
+  private readonly _startPosition: THREE.Vector3;
+  private _measurement: Measurement;
+  private _pointSize: number;
+
+  private readonly _handleonPointerClick = this.onPointerClick.bind(this);
+  private readonly _handleonPointerMove = this.onPointerMove.bind(this);
+
+  constructor(viewer: Cognite3DViewer) {
+    this._viewer = viewer;
+    this._measurementGizmo = new MeasurementGizmo(this._viewer);
+    this._measurementLabel = new MeasurementLabel(this._viewer);
+    this._startPosition = new THREE.Vector3();
+    this._pointSize = 0.02;
+  }
+
+  /**
+   * Set input handling
+   */
+  private setupEventHandling() {
+    this._viewer.on('click', this._handleonPointerClick);
+  }
+
+  /**
+   * Remove input handling
+   */
+  private removeEventHandling() {
+    this._viewer.off('click', this._handleonPointerClick);
+  }
+
+  /**
+   * Add an measurement to the control
+   * @param measurement Measurement object which is the active measurement type
+   */
+  add(measurement: Measurement): void {
```

Comment:
Who calls this? Why is it necessary to provide an instance of 'Measurement'? Is this `startMeasurement()`?